lib/all_unique.js

```diff
@@ -1,14 +1,20 @@
 module.exports = {
   all_unique: function(str){
-    // Your code goes here.
+    var abc_string = str.sort();
```

Comment:
In JavaScript, you cannot call `.sort()` on a String. You _can_ call it on an Array. I think combining the work you are doing in lines 3 and 4 will help.